docs/tutorials/digital-twin-simulation/awsim-tutorial.md

```diff
@@ -0,0 +1,106 @@
+# AWSIM simulator setup
+
+## Preparation
+
+- Install Autoware following the [official guide](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/)
+- Download and extract the [AWSIM binary](https://drive.google.com/file/d/1jgB_A5cSso9dzcNffQYthe07GG07QLQh/view?usp=sharing)
+- Download and extract the [sample map folder](https://drive.google.com/file/d/1vGFI0o0zQ-gRZYqKrPbnrtCN3c3-92Fy/view?usp=sharing)
+
+## Running steps
+
+### AWSIM Simulator
+
+To run the AWSIM simulator, open a terminal window, navigate to the simulation directory and run:
+
+```bash
+source /opt/ros/galactic/setup.bash
+cd <PATH_TO_EXTRACTED_FOLDER_WITH_AWSIM>
+./AWSIM.x86_64
+```
+
+If the simulator is working properly, Lidar rays should be visible and the [expected data topics](https://github.com/autowarefoundation/autoware-projects/wiki/Bus-ODD-Simulation-requirements#integration-interface-with-autoware-universe) should appear when `ros2 topic list` is run.
+
+![correctly working simulator](./images/workingSim.png)
+
+#### Simulator shortcuts
+
+It is possible to adjust/control the simulation through the shortcuts listed below:
+
+| Shortcut | Description                                                |
+| -------- | ---------------------------------------------------------- |
+| v        | turn on/off lidar points visualization                     |
+| m        | enter/exit manual operation mode (Ego listens to keyboard) |
+|          | <b> In manual mode: </b>                                   |
+| d        | drive                                                      |
+| p        | parking brake                                              |
+| r        | reverse                                                    |
+| arrows   | acceleration and steering                                  |
+| 1        | left blinker                                               |
+| 2        | right blinker                                              |
+| 3        | emergency lights                                           |
+| 4        | turning the lights off                                     |
```

Comment:
- Suggestion to separate out the commands from non-manual mode and manual mode
- Do the light controls (left/right blinker, emergency lights) act as toggles? So, pressing `1` turns on the left blinker and it will stay on until `1` is pressed again? Or does `4` turn the blinker off?

```suggestion
| Shortcut | Description                                                |
| -------- | ---------------------------------------------------------- |
| v        | Toggle lidar points visualization                          |
| m        | Toggle manual operation mode (Ego listens to keyboard)     |

<b> In manual mode: </b>
| Shortcut | Description                                                |
| -------- | ---------------------------------------------------------- |
| d        | Shift gear to Drive                                                      |
| p        | Shift gear to Park                                                       |
| r        | Shift gear to Reverse                                                    |
| ← ↑ ↓ → | Accelerate/brake and steering                                  |
| 1        | Left blinker                                               |
| 2        | Right blinker                                              |
| 3        | Emergency lights                                           |
| 4        | Turn off lights                                            |
```